tests/zenko_tests/node_tests/backbeat/LifecycleUtility.js

```diff
@@ -0,0 +1,91 @@
+const async = require('async');
+
+const { scalityS3Client, awsS3Client } = require('../s3SDK');
+const ReplicationUtility = require('./ReplicationUtility');
+
+class LifecycleUtility extends ReplicationUtility {
+    constructor(s3) {
+        super(s3);
+    }
+
+    deleteAllObjects(bucketName, cb) {
```

Comment:
I think you can use `ReplicationUtility.deleteAllVersions()` even if the bucket is not versioned.

Reply:
Indeed I can! Updated.